scarb/src/compiler/plugin/mod.rs

```diff
@@ -29,8 +30,8 @@ pub fn fetch_cairo_plugin(package: &Package, ws: &Workspace<'_>) -> Result<()> {
     assert!(package.is_cairo_plugin());
     let target = package.fetch_target(&TargetKind::CAIRO_PLUGIN)?;
     let props: CairoPluginProps = target.props()?;
-    // No need to fetch for buildin plugins.
-    if !props.builtin {
+    // No need to fetch for builtin or prebuilt plugins.
+    if !props.builtin && !package.is_prebuilt() {
```

Comment:
Please add a comment saying, it will not run fetch for plugins with invalid prebuilts, even though they will need to compile with Cargo before being used.